### PR TITLE
Put constraints on class length

### LINKED ISSUE #274: [PE-D][Tester A] No module title name limit
Users can input a module name that is extremely long which is not realistic as most module code are only 6 characters long. 

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: Dilysss/ped#3

### PR DESCRIPTION
**Pull request by optionalemon**

### CODE REVIEW

**Review by Bubbl3T — APPROVED** [review]
LGTM!

File: src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java [hunk 1]
```diff
@@ -20,7 +20,8 @@
  */
 public class ModuleClass implements Identity<ModuleClass>, Comparable<ModuleClass> {
 
-    public static final String MESSAGE_CONSTRAINTS = "Class names should be alphanumeric.";
+    public static final String MESSAGE_CONSTRAINTS = "Class names should be alphanumeric and"
+            + " not exceeding 25 characters.";
```
Comment (Berted) [inline src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java]:
```suggestion
            + " doesn't exceed 25 characters.";
```

**Review by Berted — CHANGES_REQUESTED** [review]
Nice! Minor nit on grammar

File: src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java [hunk 2]
```diff
@@ -20,7 +20,8 @@
  */
 public class ModuleClass implements Identity<ModuleClass>, Comparable<ModuleClass> {
 
-    public static final String MESSAGE_CONSTRAINTS = "Class names should be alphanumeric.";
+    public static final String MESSAGE_CONSTRAINTS = "Class names should be alphanumeric and"
```
Comment (optionalemon) [inline src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java]:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Class name should be alphanumeric and"
```

**Review by optionalemon — COMMENTED** [review]
Minor grammar issue

**Review by Berted — APPROVED** [review]
LGTM!

**Pull request merged**